Code/Tools/AssetProcessor/Platform/Mac/gui_info.plist

```diff
@@ -11,7 +11,7 @@
 	<key>CFBundleSignature</key>
 	<string>ASPR</string>
 	<key>CFBundleExecutable</key>
-	<string>AssetProcessor</string>
+	<string>AssetProcessorDummy</string>
```

Comment:
same here, isnt this needed by the AP?

Reply:
Same as above for editor.

Reply:
Yeah for the dummy implementation, having cmake fill in some info, like even just the target name of the real target, during the build time would be a way to solve this. It's easier to maintain one copy of this. I'll merge the cmake changes and the cpp implementation of this dummy launcher(maybe pick a better name even :P ) in another PR. I'll open a GitHub issue for that.